src/sql/parse.py

```diff
@@ -217,9 +228,47 @@ def without_sql_comment(parser, line):
     return " ".join(result)
 
 
+def split_args_and_sql(line):
+    """Separates line into args and sql query
+
+    The argparser expects - to precede an argument, but postgreSQL
+    and duckDB allow for -> and ->> to be used as JSON operators.
+    This function splits the line into two - args and sql.
+    This way we can only pass the args into the argparser, and
+    add in the sql later.
+
+    :param line: A line of SQL, preceded by option/argument strings
+    :type line: str
+    """
+    arg_line, sql_line = line, ""
+    # Only separate when json arrow operators used
+    # Otherwise, behavior is unchanged
+    if not any(op in line for op in ["->", "-->"]):
```

Comment:
some whitespace will help with readability here (e.g., line break after the if block and after the for block)

Reply:
Included more [whitespace for readability](https://github.com/ploomber/jupysql/pull/938/files#diff-22cfc6ec3e879941494b36e0dffaf24c100ab50173c413d821ebab94d0627ddfR263-R278)